src/App.js

```diff
@@ -86,12 +88,33 @@ function Nav() {
   const pfp = auth.currentUser.photoURL;
   const name = auth.currentUser.displayName;
 
-  return (
-    <nav>
-      <img alt="pfp.png" src={pfp}></img>
-      <h3>Logged in as: {name}</h3>
-    </nav>
-  );
+  if(maintainers.includes(vars.user)) {
+    return (
+      <nav>
+        <img alt="pfp.png" src={pfp}></img>
+        <h3>Logged in as: {name} (@is-a-dev/maintainers)</h3>
```

Comment:
You can get this by querying the org teams for is-a-dev. Might be an overkill but you won't have to change the code for adding/removing a new helper/maintainer

Reply:
The only thing with querying the API for this is it might add more load time to the page. As the site can run slow on older devices. Also unless we add something useful to this it will just be a bit of a gimmick for the maintainers.

Reply:
I've planned it however I've got no idea on how to write it, basically we setup a react router, and if a user is a maintainer, then we simply give them permissions to edit PRs from their, and if a user is a helper they should be able to see a list of issues they can help user's with, I've already allowed the helpers to manage help on discord and issues on GH, this would probably be a lot of work however I think it's worth it, were already planning to allow user's to edit their domains why not then

Reply:
Sorry for the spelling mistakes autocorrect ruins everything

Reply:
> it might add more load time to the page
Aren't we already querying for the user? Github's graphql api can be batched together with no effort.

I think this pr is already doing too many different things so it's alright. Always better to have 5 prs doing 1 thing each than 1 pr doing 5 things.

Anyways, good job everyone with this project. It's doing pretty well!

Reply:
> I've planned it however I've got no idea on how to write it, basically we setup a react router, and if a user is a maintainer, then we simply give them permissions to edit PRs from their, and if a user is a helper they should be able to see a list of issues they can help user's with, I've already allowed the helpers to manage help on discord and issues on GH, this would probably be a lot of work however I think it's worth it, were already planning to allow user's to edit their domains why not then

As for the issues page and the react router that would be easy to do. It's just on how you would like it to be displayed

Reply:
You just work on the functionality for that in a seperate PR, and i'll work on finishing my responsive CSS in this PR, and will merge the PR, after one final deployment of it.